File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/AutentifikacijaResultVM.cs
using Ordinacija_API.Models;

namespace Ordinacija_API.ModelVM
{
    public class AutentifikacijaResultVM
    {

        public int KorisnikID { get; set; }
        public string KorisnickoIme { get; set; }
        public string LozinkaSalt { get; set; }
        public string Telefon { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Email { get; set; }
        public string Token { get; set; }
        public Uloge Uloga { get; set; }

    }
}

File: app/src/main/java/com/example/ordinacija/Data/AutentifikacijaResultVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;

public class AutentifikacijaResultVM implements Serializable
{
    public int KorisnikID;
    public String Ime ;
    public String Prezime ;
    public String KorisickoIme ;
    public String LozinkaSalt ;
    public String Telefon ;
    public String Email ;
    public String Token;
    public int Uloga;
}

